helpers/all.js

```diff
@@ -0,0 +1,52 @@
+var _ = require('lodash'),
+    internals = {};
+
+/**
+ * Yield block only if all arguments are valid
+ *
+ * @example
+ * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/all}}
+ */
+
+internals.implementation = function(handlebars) {
+    this.handlebars = handlebars;
+};
+
+internals.implementation.prototype.register = function(context) {
+    this.handlebars.registerHelper('all', function () {
+
+        var args = [], opts, result;
+
+        // Translate arguments to array safely
+        for (var i = 0; i < arguments.length; i++) {
+            args.push(arguments[i]);
+        }
+
+        // Take the last argument (content) out of testing array
+        opts = args.pop();
+
+        // Check if all the arguments are valid / truthy
+        result = _.all(args, function(arg) {
```

Comment:
No need, but it would be nice to cut down on use of lodash if it really isn't needed. Again, nothing that needs to be done here.